packages/pulpcore/subscription-manager/subscription-manager.spec

```diff
@@ -0,0 +1,5935 @@
+#Requires: python3-dnf-plugins-core
+#Requires: python3-librepo
+%else
+Requires: python2-dnf-plugins-core
+%if (0%{?rhel} == 7)
+Requires: python-librepo
+%else
+Requires: python2-librepo
+%endif
+%endif
+#Requires: dnf >= 1.0.0
+
+%description -n dnf-plugin-subscription-manager
+This package provides plugins to interact with repositories and subscriptions
+from the Red Hat entitlement platform; contains subscription-manager and
+product-id plugins.
+%endif
+
+
+%if %use_firstboot
+%package -n subscription-manager-firstboot
+Summary: Firstboot screens for subscription manager
+Group: System Environment/Base
+Requires: rhsm-gtk = %{version}-%{release}
+Requires: rhn-setup-gnome
+
+# Fedora can figure this out automatically, but RHEL cannot:
+Requires: librsvg2
+
+%description -n subscription-manager-firstboot
+This package contains the firstboot screens for subscription-manager.
+%endif
+
+
+%if %use_initial_setup
+%package -n subscription-manager-initial-setup-addon
+Summary: initial-setup screens for subscription-manager
+Group: System Environment/Base
+Requires: rhsm-gtk = %{version}-%{release}
+Requires: initial-setup-gui >= 0.3.9.24-1
+Obsoletes: subscription-manager-firstboot < 1.15.3-1
+%if (0%{?rhel} >= 8)
+Supplements: initial-setup-gui
+%endif
+
+%description -n subscription-manager-initial-setup-addon
+This package contains the initial-setup screens for subscription-manager.
+%endif
+
+
+%if %has_ostree
+%package -n subscription-manager-plugin-ostree
+Summary: A plugin for handling OSTree content.
+Group: System Environment/Base
+
+#Requires: %{py_package_prefix}-gobject-base
+# plugin needs a slightly newer version of python-iniparse for 'tidy'
+#Requires:  %{py_package_prefix}-iniparse >= 0.4
+Requires: %{name} = %{version}-%{release}
+
+%description -n subscription-manager-plugin-ostree
+Enables handling of content of type 'ostree' in any certificates
+from the server. Populates /ostree/repo/config as well as updates
+the remote in the currently deployed .origin file.
+%endif
+
+
+%package -n %{rhsm_package_name}
+Summary: A Python library to communicate with a Red Hat Unified Entitlement Platform
+%if 0%{?suse_version}
+Group: Development/Libraries/Python
+%else
+Group: Development/Libraries
+%endif
+
+%if %use_m2crypto
+Requires: %{?suse_version:python-m2crypto} %{!?suse_version:m2crypto}
+%endif
+%if 0%{?suse_version} >= 1500
+Requires:  %{py_package_prefix}-python-dateutil
+%else
+Requires: python36-dateutil
+%endif
+Requires: python36-iniparse
+# rpm-python is an old name for python2-rpm but RHEL6 uses the old name
+Requires: %{py_package_prefix}-six
+Requires: subscription-manager-rhsm-certificates
+# Required by Fedora packaging guidelines
+%{?python_provide:%python_provide %{py_package_prefix}-rhsm}
+%if %{with python3}
+Requires: python3-rpm
+Provides: python3-rhsm = %{version}-%{release}
+Obsoletes: python3-rhsm <= 1.20.3-1
+Provides: python-rhsm = %{version}-%{release}
+Obsoletes: python-rhsm <= 1.20.3-1
```

Comment:
I don't think we should provide/obsolete these, given that on EL7 that are py2 versions.

Reply:
fixed!